### PR TITLE
feat: add hh_node measurement

### PR DESCRIPTION
**Pull request by gwossum**
Add documentation for `hh_node` measurement that was added to `/debug/vars` starting with 1.9.8 and 1.10.0.

refs https://github.com/influxdata/plutonium/pull/3877
refs https://github.com/influxdata/plutonium/pull/3878
refs https://github.com/influxdata/plutonium/pull/3879

- [x] Rebased/mergeable

### CODE REVIEW

File: content/platform/monitoring/influxdata-platform/tools/measurements-internal.md [hunk 1]
```diff
@@ -427,6 +429,26 @@ The total number of points enqueued into the hinted handoff queue.
...
+
+The `hh_node` measurement has one additional tag:
+
+- `node` - The destination node for the recorded metrics.
+
+
+#### queueTotalSize
+The sum total number of bytes of disk space used by all hinted handoff queues for a single node.
+This statistics tracks the disk usage of all hinted handoff queues for a given node, not the number
+of bytes waiting to be processed. Due to the implementation of the hinted handoff queue, there will
```
Comment (kelseiv) [inline content/platform/monitoring/influxdata-platform/tools/measurements-internal.md:444]:
```suggestion
Tracks the disk usage of all hinted handoff queues for a given node (not the bytes waiting to be processed). Due to the implementation of the hinted handoff queue, 
```

File: content/platform/monitoring/influxdata-platform/tools/measurements-internal.md [hunk 2]
```diff
@@ -427,6 +429,26 @@ The total number of points enqueued into the hinted handoff queue.
...
+be a lag between the bytes being processed and when they are removed from the disk.
```
Comment (kelseiv) [inline content/platform/monitoring/influxdata-platform/tools/measurements-internal.md:445]:
```suggestion
a lag occurs between when bytes are processed and when they're removed from the disk.
```

File: content/platform/monitoring/influxdata-platform/tools/measurements-internal.md [hunk 3]
```diff
@@ -427,6 +429,26 @@ The total number of points enqueued into the hinted handoff queue.
...
+
+`queueTotalSize` is the metric used to determine if a node's hinted handoff queue has reached the
+configured maximum size as given by the [[hinted-handoff]max-size configuration parameter](https://docs.influxdata.com/enterprise_influxdb/v1.9/administration/configure/config-data-nodes/#max-size).
```
Comment (kelseiv) [inline content/platform/monitoring/influxdata-platform/tools/measurements-internal.md:448]:
```suggestion
`queueTotalSize` is used to determine when a node's hinted handoff queue has reached the
maximum size configured in the [hinted-handoff max-size](https://docs.influxdata.com/enterprise_influxdb/v1.9/administration/configure/config-data-nodes/#max-size) parameter.
```

**Review by kelseiv — APPROVED** [review]

File: content/platform/monitoring/influxdata-platform/tools/measurements-internal.md [hunk 4]
```diff
@@ -427,6 +429,25 @@ The total number of points enqueued into the hinted handoff queue.
 
 ---
 
+### hh_node (Enterprise only)
+The `hh_node` measurement stores hinted handoff statistics for all queues (shards) for a given node.
+The `hh_node` measurement is available in InfluxDB Enterprise 1.9.8 and later.
```
Comment (kelseiv) [inline content/platform/monitoring/influxdata-platform/tools/measurements-internal.md:434]:
```suggestion
Available in InfluxDB Enterprise 1.9.8 and later.
The `hh_node` measurement stores hinted handoff statistics for all queues (shards) for a given node.

```

**Review by samhld — APPROVED** [review]

File: content/platform/monitoring/influxdata-platform/tools/measurements-internal.md [hunk 5]
```diff
@@ -427,6 +429,26 @@ The total number of points enqueued into the hinted handoff queue.
...
+
+The `hh_node` measurement has one additional tag:
+
+- `node` - The destination node for the recorded metrics.
```
[the remark this answers is not in the document]
Comment (kelseiv) [reply content/platform/monitoring/influxdata-platform/tools/measurements-internal.md:438]: @samhld I'm not clear on action item here, can you expand? thank you!

[the remark this answers is not in the document]
Comment (kelseiv) [reply content/platform/monitoring/influxdata-platform/tools/measurements-internal.md:438]: @samhld ah, I see what you mean...I'll add a docs issue

**Review by sanderson — APPROVED** [review]

**Pull request merged**